products/surface.md

```diff
@@ -23,6 +23,18 @@ auto:
 
 # Most models can be found on https://support.microsoft.com/surface/find-out-which-surface-model-you-have-da204261-8d26-9351-8588-5b09632c9486
 releases:
+-   releaseCycle: "pro-12-inch"
+    releaseLabel: "Surface Pro 12-inch"
+    releaseDate: 2025-05-20
+    eol: 2031-05-20
+    link: https://support.microsoft.com/en-us/surface/surface-pro-12-inch-features-f1ec3085-a88a-4c52-abfb-cd75f8c4e736
```

Comment:
```suggestion
    link: https://support.microsoft.com/surface/surface-pro-12-inch-features-f1ec3085-a88a-4c52-abfb-cd75f8c4e736
```

Reply:
Do not hesitate to apply those change yourself.